### PR TITLE
VEN-751 | Update layout to match design in Abstract

### PR DESCRIPTION
**Pull request by TatuArvela**
## Description 🖊️

* Fix footer to match design
* Fix navbar links to match design
* Make layout fill viewport

## Closes 🚪

* [VEN-751](https://helsinkisolutionoffice.atlassian.net/browse/VEN-751): Update layout to match design in Abstract

### CODE REVIEW

File: src/components/layout/navbar/navbar.scss [hunk 1]
```diff
@@ -31,26 +27,38 @@ $navbarTitleColor: rgba(0, 0, 0, 0.9);
...
+  }
+
+  &__title {
+    margin-left: 0.5em;
+    font-weight: bold;
+  }
+
+  &__link {
     margin-right: 1em;
+    padding: 1em 0.5em;
```
Comment (Nizar-Rahme) [inline src/components/layout/navbar/navbar.scss:47]: Can we use our predefined values, i.e. `$spacing-00-50`and `$spacing-01`?

File: src/components/layout/navbar/navbar.scss [hunk 2]
```diff
@@ -31,26 +27,38 @@ $navbarTitleColor: rgba(0, 0, 0, 0.9);
...
-    &--active {
-      @include underline($hel-black, true);
+    &:hover {
+      text-decoration: none;
     }
 
-    &--main span {
-      font-weight: bolder;
+    &--active {
+      padding-bottom: 0.7em;
```
Comment (Nizar-Rahme) [inline src/components/layout/navbar/navbar.scss:60]: And here

**Review by Nizar-Rahme — DISMISSED** [review]
header

IMHO, it looks better if we only have one line under the links, I suggest replacing the black line with the blue one after applying its animation.
And it might be unnecessary to underline the `Venepaikat` link on the uppermost header f

**Review by ahie — APPROVED** [review]
LGTM. One prettier error still causing the build to fail. Looks like Nizar's comments have been addressed or are outdated.

**Pull request merged**